Dockerfile.php7.1-busybox

```diff
@@ -0,0 +1,129 @@
+FROM ubuntu:xenial as builder
```

Comment:
If you make PHP an ARG I think the 7.1 and 7.0 files can be combined and you just build it with the different argument. Unless there are some changes between them, but they look the same.